### PR TITLE
build(dockerfile): Build Central and Central-DB for power and arm

### PR DESCRIPTION
**Pull request by connorgorman**
## Description

Testing

## Checklist
- [ ] Investigated and inspected CI test results
- [ ] Unit test and regression tests added
- [ ] Evaluated and added CHANGELOG entry if required
- [ ] Determined and documented upgrade steps
- [ ] Documented user facing changes (create PR based on [openshift/openshift-docs](https://github.com/openshift/openshift-docs) and merge into [rhacs-docs](https://github.com/openshift/openshift-docs/tree/rhacs-docs))

If any of these don't apply, please comment below.

## Testing Performed

TODO(replace-me)
Use this space to explain how you tested your PR, or, if you didn't test it, why
you did not do so. Valid reasons include, for example, "CI is sufficient",
"No testable changes". Feel free to attach JSON snippets, curl commands,
screenshots.

In addition to reviewing your code, reviewers **must** also review your testing
instructions and make sure they are sufficient.

### CODE REVIEW

File: .github/workflows/build.yaml [hunk 1]
```diff
@@ -28,24 +28,31 @@ jobs:
         run: |
           source './scripts/ci/lib.sh'
 
-          matrix='{"pre_build_go_binaries": {"name":[]}, "build_and_push_main": {"name":[]} }'
+          matrix='{ "pre_build_go_binaries": { "name":[], "arch":[] }, "build_and_push_main": { "name":[], "arch":[] }, "build_and_push_main_manifests": { "name":[] } }'
```
Comment (gavin-stackrox) [inline .github/workflows/build.yaml:31]: This job is really just about pushing the multiarch manifests, amiright?
```suggestion
          matrix='{ "pre_build_go_binaries": { "name":[], "arch":[] }, "push_main_multiarch_manifests": { "name":[], "arch":[] }, "build_and_push_main_manifests": { "name":[] } }'
```

File: scripts/ci/lib.sh [hunk 2]
```diff
@@ -157,15 +157,50 @@ get_central_diagnostics() {
     ls -l "${output_dir}"
 }
 
+push_image_manifest_lists() {
+    info "Pushing main, roxctl, central-db, scanner, collector images as manifests"
```
Comment (gavin-stackrox) [inline scripts/ci/lib.sh:161]:
```suggestion
    info "Pushing main, roxctl & central-db images as manifests"
```

**Review by gavin-stackrox — APPROVED** [review]

File: scripts/ci/lib.sh [hunk 3]
```diff
@@ -299,6 +335,11 @@ push_matching_collector_scanner_images() {
         fi
     }
 
+    if [[ "$arch" != "amd64" ]]; then
+        echo "Skipping rebundling for non-amd64 arch"
```
Comment (robbycochran) [inline scripts/ci/lib.sh:339]: Collector images are available for power, but we will need to add arm

Comment (connorgorman) [reply scripts/ci/lib.sh:339]: Yeah will need to fix this, if it's alright, I'd like to fix it in a later PR

**Review by robbycochran — APPROVED** [review]

File: scripts/ci/lib.sh (hunk 3 above)
Comment (robbycochran) [reply scripts/ci/lib.sh:339]: sounds good

**Pull request merged**